core/src/main/java/com/microsoft/applicationinsights/internal/channel/common/TransmissionNetworkOutput.java

```diff
@@ -174,28 +183,32 @@ public boolean sendSync(Transmission transmission) {
                     // to be throttled
                     transmissionPolicyManager.clearBackoff();
                 }
+                successCounter.incrementAndGet();
                 return true;
 
             } catch (ConnectionPoolTimeoutException e) {
-                ex = e;
```

Comment:
Messages for each of these exceptions is different. Hence couldnot apply the rethrowing logic here.